VoodooPS2Trackpad/VoodooPS2Trackpad-Info.plist

```diff
@@ -214,6 +214,12 @@
 					<integer>1</integer>
 					<key>ForceTouchPressureThreshold</key>
 					<integer>100</integer>
+					<key>ForceTouchCustomDownThreshold</key>
+					<integer>90</integer>
+					<key>ForceTouchCustomUpThreshold</key>
+					<integer>90</integer>
```

Comment:
Should be 20? With both values at 90 there's division by zero.

Reply:
Yes, it should